tutorial/consuming_packages/different_configurations.rst

```diff
@@ -202,6 +204,23 @@ executable can't find the shared libraries for *Zlib* that we just installed.
     $ Release\compressor.exe
     (on a pop-up window) The code execution cannot proceed because zlib1.dll was not found. Reinstalling the program may fix this problem.
 
+.. note:: **Possible issue with Windows consoles**
```

Comment:
This reads a bit too heavy for the likelihood of impact and scope, only Windows and only for shared in some terminals:
I'll try to lighten it a bit, make it much shorter. The thing is that if you activate the Conan virtualenv it should take the Conan one, so this applies only to:
- Windows user
- With a terminal like cmder that contains some zlib.dll
- That didn't activate the ``conanrun.bat`` virtualenv
- And even if they didn't, it still worked, because it found zlib.dll in the path thanks to cmder

Is this correct?

Reply:
Yes, it's true it's not as common as I thought. I think we could have a shorter notice saying that it can still run if the library is obtained from a different path by the system. It's a more generic case that can cover a wider range.

Reply:
Sounds good!